.github/scripts/addUnblockedComment.js

```diff
@@ -0,0 +1,56 @@
+// When a blocking issue is closed:
+// 1. Leaves a comment on all the issues listed as blocked in body,
+
+module.exports = async ({ github, context }) => {
+  const { ISSUE_VERIFIERS } = process.env;
+  const issueBody = context.payload.issue.body;
+  const blockedIssuesRegex = /Blocked issues:\s*(#\d+(?:,\s*#\d+)*)/i;
+
+  if (!issueBody) {
+    console.log("No issue body was found");
+    return;
+  }
+
+  // Get the list of issues blocked by this issue if there are any
+  const blockedIssuesLine = issueBody.match(blockedIssuesRegex);
+
+  // Add a "@" character to notify the user
+  const verifiers = ISSUE_VERIFIERS.split(",").map((v) => " @" + v.trim());
+
+  // If "Blocked issues" line is matched in the body then create a comment on each issue listed
+  if (blockedIssuesLine) {
+    const issueNumbers = blockedIssuesLine[0]
+      .slice(16)
+      .split(", ")
+      .map((number) => number.slice(1));
```

Comment:
The regex isn't strict enough for this match parsing. For example, the regex would match this string, which would break the slice logic:

> Blocked issues:           #123,#321

Reply:
Nice! tweaked a bit to `/(?!Blocked issues:\s)(#\d+)/gi` so it doesn't include the commas at the end of the numbers ["#123","#321","#11"]

Reply:
Nice catch, it was late and they must have blended in with the array commas lol